data/database.sql

```diff
@@ -363,7 +363,7 @@ CREATE TABLE IF NOT EXISTS `site_scan_history` (
   CONSTRAINT `fk_site_scan_history_sites1`
     FOREIGN KEY (`sites_id`)
     REFERENCES `sites` (`id`)
-    ON DELETE NO ACTION
+    ON DELETE CASCADE
```

Comment:
Remind me what these are for again and why deletes should cascade when updates do not.